python/fedml/api/modules/model.py

```diff
@@ -262,10 +262,37 @@ def info(name):
             click.echo("Failed to query model {}.".format(name))
 
 
-def run(name, data):
-    infer_out_json = FedMLModelCards.get_instance().inference_model(name, data)
+def run(endpoint_id: str, json_string: str) -> bool:
+    api_key = get_api_key()
+    if api_key == "":
+        click.echo('''
+                Please use one of the ways below to login first:
+                (1) CLI: `fedml login $api_key`
+                (2) API: fedml.api.fedml_login(api_key=$api_key)
+                ''')
+        return False
+    infer_out_json = FedMLModelCards.get_instance().endpoint_inference_api(api_key, endpoint_id, json_string)
```

Comment:
We shouldn't have click echos in api modules. Note that there are two ways to interact with out platform:
1. CLI
2. API

If users use API, they would need to have the info in response. Can we follow the pattern launch has which is return all the info from api module and have cli module echo stuff instead of echoing from the API module itself?

Reply:
Sure, will change it.